user-management-service/app/src/main/java/com/microservices/usermanagement/dto/UserDto.java

```diff
@@ -0,0 +1,19 @@
+package com.microservices.usermanagement.dto;
+
+import com.microservices.usermanagement.entity.CardInfo;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+
+@Getter
+@Setter
+@ToString
+public class UserDto {
```

Comment:
Lets name this CreateUserDto or CreationUserParametersDto

Reply:
fixed